client/src/main/scala/org/apache/celeborn/client/CommitManager.scala

```diff
@@ -122,9 +122,9 @@ class CommitManager(appUniqueId: String, val conf: CelebornConf, lifecycleManage
                               lifecycleManager.shuffleAllocatedWorkers
                                 .get(shuffleId)
                                 .asScala
-                                .find(_._1.equals(worker))
+                                .get(worker.toUniqueId())
                                 .get
-                                ._1
+                                .workerInfo
                             val primaryIds =
```

Comment:
Given we assume it is available, we can simply make this entire code something like:
```
                         lifecycleManager.shuffleAllocatedWorkers.get(shuffleId).get(worker.toUniqueId()).workerInfo
```

Reply:
thanks, addressed